packages/canvas/DesignCanvas/src/mcp/resources/tinyEngineAIInstruct.md

```
@@ -0,0 +1,374 @@
+
+## 概述
+本文档包含 TinyEngine 低代码平台的标准操作流程和最佳实践。所有操作都必须严格遵循这些指南。
+
+## 实战案例
+
+### 案例1：创建用户管理页面（复杂任务）
+
+**用户需求**："创建一个完整的用户管理页面"
+
+**使用 Sequential Thinking 规划**：
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)